### PR TITLE
Display fees on top on recurring contribution card

### PR DESCRIPTION
**Pull request by sbinlondon**
Solves opencollective/opencollective#3290

Memo suggested we follow this design

image

I adapted it to look like this

Screenshot 2020-07-13 at 14 52 15
Screenshot 2020-07-13 at 14 52 25

### CODE REVIEW

**Comment by znarf** [comment]
I like the tooltip. "Platform fee" is not the right term, I'm not sure what's best but "the additional amount you decided to donate to Open Collective" would works better for me. @piamancini can give a final call.

In the end, I'm also not sure on mixing everything in "Amount Contributed" but it's definitely already better than not surfacing it. @piamancini ?

File: components/recurring-contributions/RecurringContributionsCard.js [hunk 1]
```diff
@@ -42,6 +43,8 @@ const RecurringContributionsCard = ({
   const isError = status === ORDER_STATUS.ERROR;
   const isActive = status === ORDER_STATUS.ACTIVE || isError;
 
+  console.log(contribution.platformFee);
+
```
Comment (kewitz) [inline components/recurring-contributions/RecurringContributionsCard.js:47]:
```suggestion
```

Comment (sbinlondon) [reply components/recurring-contributions/RecurringContributionsCard.js:47]: Oh no 🤦‍♀️

**Comment by piamancini** [comment]
"Contribution to collective + contribution to the platform"

**Pull request merged**